src/server/page.ts

```diff
@@ -103,6 +103,7 @@ const template = (
           'https://www.hedvig.com/f/62762/1920x1080/a8d806bbbf/background.png'}"
      />
      <meta name="google-site-verification" content="AZ5rW7lm8fgkGEsSI8BbV4i45ylXAnGEicXf6HPQE-Q" />
+     <link rel="canonical" href="https://hedvig.com/${locale}/new-member" />
```

Comment:
what about `/forever`? Maybe we should do something that is slightly smarter than this..? I feel also like this will be missed if we would ever do something not related to `/new-member` updating this will be forgotten. Please challenge this though :D

Reply:
From what I understood we don't want to index `/forever` either, but as you said this might not be the best way to implement this. When I think of it we should move this to `/embark/index.tsx` instead, so it will only effect the "onboarding flow" pages. But I might be misunderstanding the use of canonical links :D WDYT? @palmenhq

Reply:
Updated now, we should probably only have it on `Landing.tsx` :D